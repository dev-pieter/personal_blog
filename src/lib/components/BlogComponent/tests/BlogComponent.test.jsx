import { shallow } from 'enzyme';
import React from 'react';
import { MemoryRouter, Route } from 'react-router';

import { BlogComponent } from '..';

describe("Blog Component", () => {
  let component;
  beforeAll(() => {
    component = shallow(
      <MemoryRouter initialEntries={["/posts/6163e2f80274ee9b2b105e15"]}>
        <Route path="posts/:id">
            <BlogComponent match={null} />
        </Route>
      </MemoryRouter>
    );
  });

  it("Should match the snapshot", () => {
    expect(component).toMatchSnapshot();
  });

  it("Should render blog body", () => {
    expect(component.find('.blog-body')).not.toBeNull();
  });
});
