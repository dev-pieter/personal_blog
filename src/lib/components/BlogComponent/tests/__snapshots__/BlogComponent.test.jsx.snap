// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Blog Component Should match the snapshot 1`] = `ShallowWrapper {}`;
